PoseXYZ pose # units: m
---
sensor_msgs/Image img
bool isWrapped
float64[8] imgCornersXY # XY pairs order: bottom-left, bottom-right, top-right, top-left; units: m